Automattic/simplenote-android
# Update Password Policy

### Fix
Update the `simperium` dependency, which includes saving the user object as described in https://github.com/Simperium/simperium-android/pull/222 to close #1007.

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Tap ***Log In with Email*** button.
3. Enter email address in ***Email*** field.
4. Notice ***Log In*** button is disabled.
5. Enter incorrect password of more than four and less than eight characters in ***Password*** field.
6. Notice ***Log In*** button is enabled after four characters are input.
7. Tap ***Log In*** button.
8. Notice ***Error*** dialog without password requirements is shown.
9. Tap ***OK*** button in dialog.
10. Enter correct password of more than seven characters in ***Password*** field.
11. Notice ***Log In*** button is enabled after four characters are input.
12. Tap ***Log In*** button.
13. Notice app is successfully logged in.
14. Tap ***Recent*** button in system navigation bar.
15. Remove app from recent apps view.
16. Launch app.
17. Notice app is logged in.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
